lib/ext/heroku/command/addons.rb

```diff
@@ -251,7 +256,7 @@ def attach
       case response.status
       when 201
         display("done")
-        action("Setting #{response.body["name"]} vars and restarting #{app}") do
+        action("Setting #{response.body["name"]} and restarting #{app}") do
```

Comment:
Also, sometimes the attachment won't cause a restart. `response.body["name"]` exists regardless of whether the app will be restarted or not and whether a release will be created or not. Perhaps we need to do a `api.get_release(app, 'current')` just to see if the release number has changed?